doc/telescope.txt

```diff
@@ -798,6 +802,10 @@ builtin.lsp_implementations({opts})            *builtin.lsp_implementations()*
     Parameters: ~
         {opts} (table)  options to pass to the picker
 
+    Fields: ~
+        {jump_type} (string)  how to go to implementation if there is only one,
```

Comment:
Oh, this line seems to have 79 characters.

Reply:
something is configured wrong in docgen. Ill fix at some point in time. Thanks

Reply:
Maybe "go to" can be changed to "goto".